lib/ioki/apis/operator_api.rb

```diff
@@ -292,9 +292,10 @@ class OperatorApi
         model_class: Ioki::Model::Operator::RematchingAttempt,
         except:      [:update, :delete]
       ),
-      Endpoints::Index.new(
-        :rematching_suggestions,
-        base_path:   [API_BASE_PATH, 'products', :id, 'task_lists', :id],
+      Endpoints.custom_endpoints(
```

Comment:
In case you aren't aware, `Endpoints::ShowSingular.new` could also be used here. It's used for resources which don't have an ID on their own.

No need to change though!